src/main/webapp/js/finscholar/collegepageview/collegepageview.soy

```diff
@@ -23,43 +23,43 @@
   {@param netCostForFourthQuintile: string}
   {@param netCostForFifthQuintile: string}
   {@param cumulativeMedianDebt: string}
-  <div class="mdl-grid mdl-shadow--2dp">
+  <div class="body mdl-grid mdl-shadow--2dp">
```

Comment:
You can directly overwrite styles for mdl classes in custom css files. (You don't need to change anything)